src/System.Runtime.Extensions/tests/System/IO/PathTests_Join.netcoreapp.cs

```diff
@@ -84,6 +85,7 @@ public void TryJoinTwoPaths(string path1, string path2, string expected)
         [Theory, MemberData(nameof(TestData_JoinThreePaths))]
         public void JoinThreePaths(string path1, string path2, string path3, string expected)
         {
+            Assert.Equal(expected, Path.Join(path1.AsSpan(), path2.AsSpan(), path3.AsSpan()));
```

Comment:
We should also have tests that verify the behavior when nulls are passed in.

Reply:
Ok i'll do It!

Reply:
dont, let me know if are enough.

Reply:
Thanks. LGTM.